packages/suite/src/support/messages.ts

```diff
@@ -7565,4 +7561,13 @@ export default defineMessages({
         description: 'Button opening changelog',
         defaultMessage: 'View all',
     },
+    TR_DISABLED_ANONYMITY_CHANGE_MESSAGE: {
+        id: 'TR_DISABLED_ANONYMITY_CHANGE_MESSAGE',
+        defaultMessage: 'Editing disabled while Coinjoin is running',
+    },
+    TR_LOW_ANONYMITY_WARNING: {
+        id: 'TR_LOW_ANONYMITY_WARNING',
+        defaultMessage:
+            "<red>Very low anonymity.</red> We recommend using atleast 1 in 5, as anything bellow isn't secure.",
```

Comment:
typo - missing space
```suggestion
            "<red>Very low anonymity.</red> We recommend using at least 1 in 5, as anything bellow isn't secure.",
```

Reply:
6b84fc60c0782730d227ef3e2f51fb09ea1a7b56